help_test.go

```diff
@@ -219,3 +220,14 @@ Commands:
 		require.Equal(t, expected, w.String())
 	})
 }
+
+func TestEnvarAutoHelp(t *testing.T) {
+	var cli struct {
+		Flag string `env:"FLAG" help:"A flag."`
+	}
+	w := &strings.Builder{}
+	p := mustNew(t, &cli, kong.Writers(w, w), kong.Exit(func(int) {}))
+	_, err := p.Parse([]string{"--help"})
+	require.NoError(t, err)
+	require.Contains(t, w.String(), "A flag ($FLAG).")
+}
```

Comment:
Please add a test for custom help formatting.

Reply:
Added with the latest commit